packages/altair-fastify-plugin/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "altair-fastify-plugin",
+  "version": "2.4.10",
+  "description": "Fastify Plugin of Altair GraphQL Client",
+  "keywords": [
+    "graphql",
+    "fastify",
+    "altair",
+    "client",
+    "gql"
+  ],
+  "homepage": "https://github.com/imolorhe/altair#readme",
+  "bugs": {
+    "url": "https://github.com/imolorhe/altair/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/imolorhe/altair.git"
+  },
+  "license": "MIT",
+  "author": "PabloSz <pablosaez1995@gmail.com>",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "example"
+  ],
+  "scripts": {
+    "example": "ts-node example/fastifyGQL.ts",
+    "prepublish": "tsc"
```

```suggestion
    "test": "echo \"Error: no test specified\" && exit 0",
    "prepare": "tsc"
```

The pipeline is failing and it might be because of this.